frontend/src/components/workspaces/Templates/TemplatesTableLogic/TemplatesTableLogic.tsx

```diff
@@ -68,6 +77,9 @@ const TemplatesTableLogic: FC<ITemplateTableLogicProps> = ({ ...props }) => {
         ),
       });
     }
+    return () => {
+      unsubscribe && unsubscribe();
+    };
```

Comment:
```suggestion
    return () => unsubscribe?.();
```
and other occurrences :)

Reply:
Why there is a `.` in your suggestion?